src/NuGet.Core/NuGet.CommandLine.XPlat/ReportRenderers/Models/ListPackageProjectModel.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Build.Evaluation;
+using NuGet.CommandLine.XPlat.Utility;
+using NuGet.ProjectModel;
+
+namespace NuGet.CommandLine.XPlat
+{
+    /// <summary>
+    /// Calculated project data model for list report
+    /// </summary>
+    internal class ListPackageProjectModel
+    {
+        private const string ProjectNameMSbuildProperty = "MSBuildProjectName";
+        internal List<ReportProblem> ProjectProblems { get; } = new List<ReportProblem>();
+        internal string ProjectPath { get; private set; }
+        // Original packages
+        internal IEnumerable<FrameworkPackages> Packages { get; private set; }
+        // Calculated project model data for each targetframeworks
+        internal List<ListPackageReportFrameworkPackage> TargetFrameworkPackages { get; private set; }
+        internal string HttpSourceWarning { get; private set; }
+        internal string ProjectName { get; private set; }
+        internal Project Project { get; }
+        internal ListPackageReportModel ReportModel { get; }
```

Comment:
Doesn't having ListPackageReportModel create a circular reference? 

Why do we need this?

I'd expect the report model that contains all projects models, but we shouldn't need the reference the other way around.

Reply:
Sorry, I was lazy, it was more convenient when I am using it, I just removed.